pkg/config/config.go

```diff
@@ -129,7 +130,8 @@ type TLSConfig struct {
 type Registry struct {
 	// Mirrors are namespace to mirror mapping for all namespaces.
 	Mirrors map[string]Mirror `toml:"mirrors" json:"mirrors"`
-	// Configs are configs for each registry endpoint.
+	// Configs are configs for each registry.
+	// The key is the registry host address.
```

Comment:
host address?  is that the same as fqdn or host name?  when I think address I usually think tcip address... Guess we are picking up the naming from the auth.ServerAddress field?

Reply:
Yeah, it is the hostname/ip

Reply:
Done.